### PR TITLE
[Notifier] Add symfony/gitter-notifier docs

### PR DESCRIPTION
**Pull request by christingruber**
See https://github.com/symfony/symfony/pull/39838

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -170,6 +171,10 @@ Zulip       ``symfony/zulip-notifier``        ``zulip://EMAIL:TOKEN@HOST?channel
     The GoogleChat, LinkedIn, Zulip and Discord integrations were introduced in Symfony 5.2.
     The Slack DSN changed in Symfony 5.2 to use Slack Web API again same as in 5.0.
 
+.. versionadded:: 5.3
+
+    The Gitter integration were introduced in Symfony 5.3.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The Gitter integration was introduced in Symfony 5.3.
```

**Review by OskarStark — APPROVED** [review]
After my comment 👍🏻

**Comment by OskarStark** [comment]
Can you please rebase? Thanks

**Comment by christingruber** [comment]
Rebased :+1:

**Review by xabbuh — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Christin for working on this feature, this is much appreciated.

**Pull request merged**